config/urls.py

```diff
@@ -31,6 +31,7 @@
         ),
         # Django Admin, use {% url 'admin:index' %}
         path(settings.ADMIN_URL, admin.site.urls),
+        path(f'{settings.ADMIN_URL}/doc/', include('django.contrib.admindocs.urls')),
```

Comment:
Question: What does this do or enable?

Reply:
It enable an handi page that use the comment to write documentation about model and view.

See an intro here:
https://nearsoft.com/blog/the-secret-of-django-auto-generated-admin-docs/

I used it to see what model i should remove here: https://github.com/tychota/katago-server/commit/3adaa6de03f1cfe0fa4dec61a18c933ec94e5a79

It does work in dev and should work in prod (but isn't working because a missing deps).